docs/source/install/__packages_beta_warning.rst

```diff
@@ -0,0 +1,7 @@
+.. warning::
+
+   Packages based installation is currently is beta and upgrades are not yet
```

Comment:
I really think we should just remove the outdated methods.


On 4 Apr 2016 18:32 +0600, Tomaz Muraus<notifications@github.com>, wrote:
>  
> Indocs/source/install/__packages_beta_warning.rst(https://github.com/StackStorm/st2docs/pull/110#discussion_r58368117):
>  
> >@@ -0,0 +1,7 @@>+.. warning::>+>+  Packages based installation is currently is beta and upgrades are not yet
>  
> I'm open to wording this differently or removing this notice all together.
>  
>  
> The fact is that AIO is out of date and not supported so we need to point all the new users to the package based installation instructions and this warning might scare some people off...
>  
>  
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly orview it on GitHub(https://github.com/StackStorm/st2docs/pull/110/files/ba8ed5f6ae8f381987e3f85c57dd3f0d0271cdc1#r58368117)
>  
>  
>